compiler/passes/parallel.cpp

```diff
@@ -375,7 +375,7 @@ bundleArgs(CallExpr* fcall, BundleArgsFnData &baData) {
 
     // Now get the taskList field out of the end count.
```

Comment:
No the comment is still up-to-date.  There is still a task list, at least fifo maintains one, but it's not in a separate data structure in that tasking layer, and the compiler doesn't need to know anything about its type. Outside of the runtime all that need be known is that it must be unique for each parallel construct and that it must be big enough to hold a pointer.